website/source/api/secret/databases/mongodb.html.md

```diff
@@ -46,7 +49,10 @@ has a number of parameters to further configure a connection.
   "connection_url": "mongodb://{{username}}:{{password}}@mongodb.acme.com:27017/admin?ssl=true",
   "write_concern": "{ \"wmode\": \"majority\", \"wtimeout\": 5000 }",
   "username": "admin",
-  "password": "Password!"
+  "password": "Password!",
+  "ssl_cert": "/path/to/ssl/cert",
```

Comment:
These parameters would not be paths.  They should be PEM encoded certs.

Reply:
You are right